topics:
  input:
    image: "/anafi/image"
  output:
    water_mask: "/segmentation/dl_mask"

model_settings:
  model_path: "/home/msccomputer/catkin_ws/src/msc_thesis/train_segmentation/BEST_MODELS/14-03-2023-10-39/best_model.pth"
  classes: 1
  bilinear: False
  model_type: "segnet"
  
services:
  make_mask: make_dl_mask